p2p/p2p.proto

```diff
@@ -38,8 +38,17 @@ message AnchorDocumentResponse {
   bool accepted = 1;
 }
 
+enum AccessType {
+  ACCESS_TYPE_INVALID = 0;
+  ACCESS_TYPE_REQUESTER_VERIFICATION = 1;
+  ACCESS_TYPE_NT_OWNER_VERIFICATION = 2;
+  ACCESS_TYPE_ACCESS_TOKEN_VERIFICATION = 3;
+}
+
 message GetDocumentRequest {
   bytes document_identifier = 1;
```

Comment:
Will we only support getting the current/latest version of the document? Not passed versions ?

Reply:
this PR supports getting just the current version, I will submit future PRs that  handle specific past versions, as well as returning all versions prior to the current one (see Document List on the Read ACL spec). Just the prelim, most basic version of the GetDocumentRequest for now in this PR :)